### PR TITLE
fix travis for quebec

### PR DESCRIPTION
**Pull request by lippirk**
Testing travis with quebec

### CODE REVIEW

File: .travis-xs-opam.sh [hunk 1]
```diff
@@ -4,10 +4,10 @@ set -ex
 
 PACKAGE="xapi"
 PINS="xapi:. xapi-cli-protocol:. xapi-client:. xapi-consts:. xapi-database:. xapi-datamodel:. xapi-types:. xe:."
-BASE_REMOTE="https://github.com/xapi-project/xs-opam.git"
+BASE_REMOTE="https://github.com/xapi-project/xs-opam.git#quebec"
```
Comment (psafont) [inline .travis-xs-opam.sh]: This is already defined in the xs-opam-ci.env. the variables for setting the remote and its branch are different ones, looking at it.

**Comment by edwintorok** [comment]
If there are LCM branches for components, do we want to update xs-opam to point to the LCM branches of the component instead of a version number? Probably not worth it for Quebec, but for Stockholm which is more long lived.

**Comment by psafont** [comment]
The problem is that strategy makes us create an LCM branch for all repositories right away. We can pin it to a version initially, and when the version needs to be changed pin it to a branch instead.

**Comment by lippirk** [comment]
> If there are LCM branches for components, do we want to update xs-opam to point to the LCM branches of the component instead of a version number? Probably not worth it for Quebec, but for Stockholm which is more long lived.

i agree with @psafont that we should do it for branches only when necessary, since the vast majority of repos we will not change (unless specsavers happens soon)

**Comment by psafont** [comment]
I'd really like to have a PR for the xs-opam branch before merging this one

**Comment by lippirk** [comment]
> I'd really like to have a PR for the xs-opam branch before merging this one

done - i'll update this according to those changes

**Pull request merged**